src/assets/translations/en/main.json

```diff
@@ -2191,16 +2194,53 @@
     "poolInformation": "Pool Information",
     "poolsBody": "Provide liquidity and earn fees on trades in that pool.",
     "availablePools": "Available Pools",
+    "addLiquidity": "Add Liquidity",
+    "removeLiquidity": "Remove Liquidity",
+    "initialPricesAndPoolShare": "Initial Prices and Pool Share",
+    "pricePerAsset": "%{from} per %{to}",
+    "symAlert": "Your %{from} will be split into a 50:50 exposure to both %{to} and %{from} upon entering the pool.",
+    "shareOfPool": "Share of Pool",
+    "assetDepositAmount": "%{asset} Deposit Amount",
     "totalLiquidity": "Total Liquidity",
     "totalVolume": "Total Volume",
     "fees24h": "Fees 24H",
-    "addLiquidity": "Add Liquidity",
-    "removeLiquidity": "Remove Liquidity",
+    "estimatedTime": "Estimated Time",
     "liquidityValue": "Liquidity Value",
+    "selectPair": "Select Pair",
+    "depositAmounts": "Deposit Amounts",
+    "awaitingConfirmation": "Awaiting confirmation",
+    "transactionSubmitted": "Transaction submitted",
+    "waitingForConfirmation": "Waiting for Confirmation",
+    "chainFee": "%{chain} Fee",
+    "supplying": "Supplying",
+    "confirmAndDeposit": "Confirm and Deposit",
     "yourPositions": {
       "yourPositions": "Your Positions",
       "emptyTitle": "No active positions",
       "emptyBody": "Your active liquidity pool positions will appear here."
+    },
+    "faq": {
+      "title": "FAQ",
+      "what": {
+        "title": "What is a Liquidity Pool?",
+        "body": "A liquidity pool is a decentralized fund pool that contains assets used to facilitate trading on a blockchain network. It plays a crucial role in decentralized finance (DeFi) by providing liquidity for various trading pairs."
+      },
+      "how": {
+        "title": "How do Liquidity Pools work?",
+        "body": "Liquidity pools work on the principle of users depositing their assets into a smart contract to create a pool. These assets are then utilized to enable decentralized exchanges, allowing users to trade against the pool. In return for providing liquidity, users receive a share of the trading fees generated by the platform."
```

Comment:
Could be made more THOR-specific, there are no smart contracts per se on THORChain (there may be if you deposit from an EVM chain using the router contract, but the funds aren't held in a "smart contract" per se, pools are network primitives)